api/v2.0/swagger.yaml

```diff
@@ -2343,6 +2343,320 @@ paths:
+          required: true
+          schema:
+            type: object
+            properties:
+              action:
+                type: string
+      responses:
+        '200':
+          description: Stop a Retention job successfully.
+        '401':
+          description: User need to log in first.
+        '403':
+          description: User have no permission.
+        '500':
+          description: Unexpected internal errors.
+
+  /retentions/{id}/executions/{eid}/tasks:
+    get:
+      summary: Get Retention job tasks
+      description: Get Retention job tasks, each repository as a task.
+      tags:
+        - Retention
+      parameters:
+        - name: id
+          in: path
+          type: integer
+          format: int64
+          required: true
+          description: Retention ID.
+        - name: eid
+          in: path
+          type: integer
+          format: int64
+          required: true
+          description: Retention execution ID.
+        - name: page
+          in: query
+          type: integer
+          format: int64
+          required: false
+          description: The page number.
+        - name: page_size
+          in: query
+          type: integer
+          format: int64
+          required: false
+          description: The size of per page.
+      responses:
+        '200':
+          description: Get Retention job tasks successfully.
+          schema:
+            type: array
+            items:
+              type: object
+              $ref: '#/definitions/RetentionExecutionTask'
+          headers:
+            X-Total-Count:
+              description: The total count of available items
+              type: integer
+            Link:
+              description: Link to previous page and next page
+              type: string
+        '401':
+          description: User need to log in first.
+        '403':
+          description: User have no permission.
+        '500':
+          description: Unexpected internal errors.
+
+  /retentions/{id}/executions/{eid}/tasks/{tid}:
+    get:
+      summary: Get Retention job task log
+      description: Get Retention job task log, tags ratain or deletion detail will be shown in a table.
+      tags:
+        - Retention
+      parameters:
+        - name: id
+          in: path
+          type: integer
+          format: int64
+          required: true
+          description: Retention ID.
+        - name: eid
+          in: path
+          type: integer
+          format: int64
+          required: true
+          description: Retention execution ID.
+        - name: tid
+          in: path
+          type: integer
+          format: int64
+          required: true
+          description: Retention execution ID.
+      responses:
+        '200':
+          description: Get Retention job task log successfully.
+          headers:
+            Content-Type:
+              description: The content type of response body
+              type: string
```

Comment:
Please use produces `text/plain` for in API and remove `Content-Type` header in its response.

See getGCLog getReportLog GetPreheatLog getReplicationLog for more info.